<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvnPropSetDialog</class>
 <widget class="QDialog" name="SvnPropSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Subversion Property</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Property Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="propNameEdit">
       <property name="toolTip">
        <string>Enter the name of the property to be set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select property source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="textRadioButton">
        <property name="text">
         <string>Text</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="propTextEdit">
        <property name="toolTip">
         <string>Enter text of the property</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fileRadioButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="E5PathPicker" name="propFilePicker" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Enter the name of a file for the property</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>propNameEdit</tabstop>
  <tabstop>textRadioButton</tabstop>
  <tabstop>propTextEdit</tabstop>
  <tabstop>propFilePicker</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>textRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>propTextEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SvnPropSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SvnPropSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>propFilePicker</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
